src/Pages/Profile/MemberView/PrintRequest.js

```diff
@@ -15,42 +15,41 @@ export default function PrintRequest(props) {
   }
 
   return (
-    <Button
-      onClick={async () => {
-        await updateRequests();
-        setToggle(!toggle);
-      }}
-      color='primary'
```

Comment:
you should revert this file to be what it is on `dev`.

```
git checkout origin/dev [filename]
```

also turn prettier off!! i know mosh uses it but it causes much pain later on